### PR TITLE
Feature to add a CSV file download route

### PR DESCRIPTION
**Pull request by bernestdev**
### General
Added a new route to handle the download of CSV files

### Type

> ℹ️  What types of changes does your code introduce?

> 👉 _Put an `x` in the boxes that apply_

- [ ] Fix
- [x] Feature

### Describe the changes here (summarized)
- Added new route `/:id/logs/download` that returns a string and the header `Content-Type: text/csv`
- Added `es6` env to the `.eslintrc.json` file to allow support for extra es6 features including the ability to create promises with `new Promise()`
- Added `csv-stringify` library for converting arrays into a string CSV

### CODE REVIEW

File: src/controllers/CheckController.js
Comment (itsalb3rt) [inline src/controllers/CheckController.js:1]: Great work! All works as expected. Only one thing, You can move the DB interaction of the `Checks` model to the `ChecksService` File, this is to keep the consistency. All the DB interactions are in services files and the controllers are only for HTTP handling and service delegation.

File: src/services/CheckService.js [hunk 1]
```diff
@@ -679,6 +681,17 @@ class CheckService {
     });
   }
 
+  static async getLogsCSV(logs) {
```
Comment (itsalb3rt) [inline src/services/CheckService.js:684]: The comment about the DB interaction can move here. ☺️

**Review by itsalb3rt — CHANGES_REQUESTED** [review]

**Comment by bernestdev** [comment]
Thank you @itsalb3rt . I made the requested changes. Please let me know if this satisfies the request

**Review by itsalb3rt — APPROVED** [review]
Awesome, work perfect!

**Pull request merged**